distribution/integ-test-zip/build.gradle

```diff
@@ -35,5 +37,14 @@ publishing {
   }
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
IMO instead of doing this we should do this in the netty4-module:

```
integTest {
  includePackaged = true
}```

that way we get the same coverage here